plugins/BEdita/API/tests/TestCase/Controller/FoldersControllerTest.php

```diff
@@ -518,20 +543,91 @@ public function setRelationshipsAllowedMethodsProvider()
      *
      * @param int $expected The expected HTTP status code.
      * @param string $method The http method.
-     * @param array|null $data Payload to use for the request.
+     * @param array $data Payload to use for the request.
      * @return void
+     * @throws \Exception
```

Comment:
[minor] Not really :smile_cat:

Reply:
Yes, you're right… 😞 It's PHPStorm that keeps on complaining about the lack of this annotation because `IntegrationTestCase::_sendRequest()` method's docblock includes `@throws \Exception`. 😡